conans/cli/commands/inspect.py

```diff
@@ -0,0 +1,31 @@
+import json
+
+from conans.cli.command import conan_command, COMMAND_GROUPS
+from conans.cli.output import ConanOutput
+from conans.errors import ConanException
+
+
+def _inspect_json_formatter(data):
+    return json.dumps(data, indent=4)
+
+
+@conan_command(group=COMMAND_GROUPS['creator'], formatters={"json": _inspect_json_formatter})
+def inspect(conan_api, parser, *args, **kwargs):
+    """
+    Returns the specified attribute/s of a conanfile
+    """
+    parser.add_argument("path", help="Path to a folder containing a recipe (conanfile.py)")
```

Comment:
Only for local conanfiles in user folder, so no possible to inspect one recipe in the cache? Will that need a custom command?

Reply:
`conan cache path` that's what we talked

Reply:
Alternatives?

`conan inspect [--path] [--reference] attr1 attr2``  => bit messy
`conan inspect path attr1 attr2`` + ``conan inspect reference attr1 attr2`` => a bit more clean

Do we see `conan inspect` as a `name` that we want to reserve for more use cases like packages?